test/parallel/test-runner-cli.js

```diff
@@ -28,10 +28,10 @@ const testFixtures = fixtures.path('test-runner');
   assert.strictEqual(child.signal, null);
   assert.strictEqual(child.stderr.toString(), '');
   const stdout = child.stdout.toString();
-  assert.match(stdout, /ok 1 - this should pass/);
-  assert.match(stdout, /not ok 2 - this should fail/);
-  assert.match(stdout, /ok 3 - .+subdir.+subdir_test\.js/);
-  assert.match(stdout, /ok 4 - this should pass/);
+  assert.match(stdout, /ok \d+ - this should pass/);
```

Comment:
I think we should keep this as numbers.
when working on https://github.com/nodejs/node/pull/47675 I see it does catch race conditions that don't exist in the snapshot tests, since they don't run a lot of files concurrently

Reply:
we can probably move all these files into a subdir